services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -204,19 +204,36 @@ private List<AbstractVeniceService> createServices() {
     jvmStats = new VeniceJVMStats(metricsRepository, "VeniceJVMStats");
 
     if (serverConfig.isSystemSchemaInitializationAtStartTimeEnabled()) {
+      String localControllerUrl = serverConfig.getLocalControllerUrl();
+      String d2ServiceName = serverConfig.getLocalControllerD2ServiceName();
+      String d2ZkHost = serverConfig.getLocalD2ZkHost();
+      String cluster = serverConfig.getSystemSchemaClusterName();
       ControllerClientBackedSystemSchemaInitializer metaSystemStoreSchemaInitializer =
           new ControllerClientBackedSystemSchemaInitializer(
               AvroProtocolDefinition.METADATA_SYSTEM_SCHEMA_STORE,
-              serverConfig.getSystemSchemaClusterName(),
+              cluster,
               AvroProtocolDefinition.METADATA_SYSTEM_SCHEMA_STORE_KEY.getCurrentProtocolVersionSchema(),
               VeniceSystemStoreUtils.DEFAULT_USER_SYSTEM_STORE_UPDATE_QUERY_PARAMS,
               true,
               sslFactory,
-              serverConfig.getLocalControllerUrl(),
-              serverConfig.getLocalControllerD2ServiceName(),
-              serverConfig.getLocalD2ZkHost(),
+              localControllerUrl,
+              d2ServiceName,
+              d2ZkHost,
+              false);
+      ControllerClientBackedSystemSchemaInitializer kmeSchemaInitializer =
+          new ControllerClientBackedSystemSchemaInitializer(
+              AvroProtocolDefinition.KAFKA_MESSAGE_ENVELOPE,
+              cluster,
+              null,
+              null,
+              false,
+              sslFactory,
+              localControllerUrl,
+              d2ServiceName,
+              d2ZkHost,
               false);
       metaSystemStoreSchemaInitializer.execute();
+      kmeSchemaInitializer.execute();
```

Comment:
Since we've initialized KME schema here, it's not necessary to verify the existence of KME schema later at line 259:
```
kafkaMessageEnvelopeSchemaReader.ifPresent(
          schemaReader -> new SchemaPresenceChecker(schemaReader, AvroProtocolDefinition.KAFKA_MESSAGE_ENVELOPE)
              .verifySchemaVersionPresentOrExit());
```
Please move above code to the following branch:
```
// For system schemas initialized via controller client, no need to verify again because they should already exist
      if (!serverConfig.isSystemSchemaInitializationAtStartTimeEnabled()) {
```